packages/web/components/place-limit-tool/index.tsx

```diff
@@ -0,0 +1,185 @@
+  ({ orderDirection = OrderDirection.Bid }) => {
+    const { accountStore } = useStore();
+    const { t } = useTranslation();
+    const [baseDenom, setBaseDenom] = useState<string>("ION");
+    const quoteDenom = "OSMO";
+
+    const { poolId, orderbookContractAddress } = useOrderbookPool({
+      baseDenom,
+      quoteDenom,
+    });
+    const swapState = usePlaceLimit({
+      osmosisChainId: "localosmosis",
+      poolId,
+      orderDirection,
+      useQueryParams: false,
+      orderbookContractAddress,
+      baseDenom,
+      quoteDenom,
+    });
+    const account = accountStore.getWallet("localosmosis");
+
+    const isSwapToolLoading = false;
+
+    return (
+      <div className="flex flex-col gap-3">
+        <TokenSelectLimit
+          selectableAssets={[swapState.fromAsset, swapState.toAsset]}
+          baseAsset={swapState.baseAsset}
+          quoteAsset={swapState.quoteAsset}
+          baseBalance={swapState.baseTokenBalance}
+          quoteBalance={swapState.quoteTokenBalance}
+          onTokenSelect={(newDenom) => setBaseDenom(newDenom)}
+          disabled={false}
+          orderDirection={orderDirection}
+        />
+        <div className="px-4 py-[22px] transition-all md:rounded-xl md:py-2.5 md:px-3">
+          <div className="flex place-content-end items-center transition-opacity">
+            <div className="flex items-center gap-1.5">
+              <Tooltip
+                content={
+                  <div className="text-center">
+                    {t("swap.maxButtonErrorNoBalance")}
+                  </div>
+                }
+                disabled={!swapState.inAmountInput.notEnoughBalanceForMax}
+              >
+                <Button
+                  variant="outline"
+                  size="sm"
+                  className={classNames(
+                    "text-wosmongton-300",
+                    swapState.inAmountInput.isMaxValue &&
+                      !swapState.inAmountInput
+                        .isLoadingCurrentBalanceNetworkFee &&
+                      !swapState.inAmountInput.hasErrorWithCurrentBalanceQuote
+                      ? "bg-wosmongton-100/20"
+                      : "bg-transparent"
+                  )}
+                  disabled={
+                    !swapState.inAmountInput.balance ||
+                    swapState.inAmountInput.balance.toDec().isZero() ||
+                    swapState.inAmountInput.notEnoughBalanceForMax
+                  }
+                  isLoading={false}
+                  loadingText={t("swap.MAX")}
+                  classes={{
+                    spinner: "!h-3 !w-3",
+                    spinnerContainer: "!gap-1",
+                  }}
+                  onClick={() => swapState.inAmountInput.toggleMax()}
+                >
+                  {t("swap.MAX")}
+                </Button>
+              </Tooltip>
+            </div>
+          </div>
+          <div className="mt-3 flex place-content-between items-center">
```

Comment:
why not? `place-content-*` only applies to grid systems, not flexboxes
```suggestion
          <div className="mt-3 flex justify-between items-center">
```

Reply:
https://github.com/osmosis-labs/osmosis-frontend/pull/3308/commits/c86e5b34d800f2435163490fac939294643f1333